src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java

```diff
@@ -156,11 +178,12 @@ private PluginSettings(String cfPath) {
     }
     LOG.info(
         "Config: metricsLocation: {}, metricsDeletionInterval: {}, httpsEnabled: {},"
-            + " cleanup-metrics-db-files: {}",
+            + " cleanup-metrics-db-files: {}, batch-metrics-retention-period: {}",
```

Comment:
Let's add the unit here as well, so that it is reflected in the logs

Reply:
Yes